src/views/Details/TransactionDetails.vue

```diff
@@ -160,7 +168,7 @@ export default {
     itemFee() {
       return typeof this.item.fee !== 'object'
         ? this.item.fee
-        : this.item.fee.suggestedBaseFeePerGas + this.item.fee.maxPriorityFeePerGas
+        : BN(this.item.fee.suggestedBaseFeePerGas + this.item.fee.maxPriorityFeePerGas).dp(3)
```

Comment:
Hi @AndonMitev, Harsh shared this and mentioned the fee calculation is in a range and not specific.

![image](https://user-images.githubusercontent.com/47292405/150128097-48c4cddb-3af6-4d3f-abc6-f44a24769407.png)

Reply:
Hi @Jennievon this is not where we set the fee, this is what fee have been set by the user and it's used to be displayed. Lower and upper bounds are used when we manipulating the custom fees